### PR TITLE
AM deprecation modifications

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/configure-monitoring-cloudhub.adoc [hunk 1]
```diff
@@ -9,6 +9,8 @@ To start monitoring Mule apps that are managed in CloudHub, you must deploy them
 
 Anypoint Monitoring with Mule runtime engine images receive patch updates as soon as they are available. Because Anypoint Monitoring -AM images are no longer receiving patch updates, switch to a Mule runtime engine (non-AM) image if your production environment requires a Mule runtime engine patch.
```
Comment (michael-lee-mulesoft) [inline modules/ROOT/pages/configure-monitoring-cloudhub.adoc]: @andrewb-ms  Should we remove or keep this part (about non-AM no longer receiving patch updates)?

File: modules/ROOT/pages/dashboard-config-ref.adoc [hunk 2]
```diff
@@ -30,7 +30,7 @@ Many <<metrics_basic, basic>> and <<metrics_advanced, advanced>> metrics are ava
 [[metrics_basic]]
 === Basic Mode Metrics for Applications
 
-These metrics are supported for the following Mule runtime engine versions: 3.8.7-AM, 3.9.x, 4.1.x, and 4.2.0.
+These metrics are supported for the following Mule runtime engine versions: 3.8.7, 3.9.x, 4.1.x, and 4.2.0.
```
Comment (michael-lee-mulesoft) [inline modules/ROOT/pages/dashboard-config-ref.adoc]: 4.2.x instead of 4.2.0

Comment (sarathecoubian) [inline modules/ROOT/pages/dashboard-config-ref.adoc]:
```suggestion
These metrics are supported for the following Mule runtime engine versions: 3.8.7, 3.9.x, 4.1.x, and 4.2.x.
```

**Comment by myuin-ms** [comment]
looks good to me

File: modules/ROOT/pages/configure-monitoring-cloudhub.adoc [hunk 3]
```diff
@@ -43,13 +45,13 @@ an appropriate value, for example:
...
- --runtime="3.9.0-AM" helloworld ~/demoproj2.zip
+ --runtime="3.9.4" helloworld ~/demoproj2.zip
 ----
 +
 * For Maven deployments:
 +
-If you are using Maven for deployment, you need to modify the `<muleVersion>`
-tag in the `pom.xml` file to append the `-AM`, for example, changing `<muleVersion>3.9.0</muleVersion>` to `<muleVersion>3.9.0-AM</muleVersion>`).
+If you are using Maven for deployment, you must modify the `<muleVersion>`
+tag in the `pom.xml` file. For example, changing `<muleVersion>3.9.3</muleVersion>` to `<muleVersion>3.9.4</muleVersion>`).
```
Comment (rddheath) [inline modules/ROOT/pages/configure-monitoring-cloudhub.adoc]: The sentence that begins with "For example" is a fragment.

File: modules/ROOT/pages/dashboard-config-ref.adoc [hunk 4]
```diff
@@ -30,7 +30,7 @@ Many <<metrics_basic, basic>> and <<metrics_advanced, advanced>> metrics are ava
 [[metrics_basic]]
 === Basic Mode Metrics for Applications
 
-These metrics are supported for the following Mule runtime engine versions: 3.8.7-AM, 3.9.x, 4.1.x, and 4.2.0.
+These metrics are supported for the following Mule runtime engine versions: 3.8.7, 3.9.x, 4.1.x, and 4.2.x.
```
Comment (rddheath) [inline modules/ROOT/pages/dashboard-config-ref.adoc]: "Mule runtime engine versions": Couldn't that noun string be simply "versions of Mule Runtime"?

File: modules/ROOT/pages/anypoint-custom-metrics-connector.adoc [hunk 5]
```diff
@@ -15,7 +15,7 @@ To see the metrics you configure in the Anypoint Custom Metrics connector in Any
 
 == Prerequisites
 
-* Mule runtime engine 4.2.x, 4.1.x, 3.9.x, or 3.8.7-AM
+* Mule runtime engine 4.2.x, 4.1.x, 3.9.x, or 3.8.7
```
Comment (rddheath) [inline modules/ROOT/pages/anypoint-custom-metrics-connector.adoc]: "Mule runtime engine" -> "Mule Runtime"

File: modules/ROOT/pages/configure-monitoring-cloudhub.adoc (hunk 3 above)
Comment (rddheath) [inline modules/ROOT/pages/configure-monitoring-cloudhub.adoc]: You're not asking the reader to modify the tag; you're asking the reader to change the value of `muleVersion`.

Comment (sarathecoubian) [reply modules/ROOT/pages/configure-monitoring-cloudhub.adoc]:
```suggestion
tag in the `pom.xml` file. For example, you can change `<muleVersion>3.9.3</muleVersion>` to `<muleVersion>3.9.4</muleVersion>`).
```

File: modules/ROOT/pages/dashboard-config-ref.adoc (hunk 4 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/dashboard-config-ref.adoc]:
```suggestion
These metrics are supported for the following versions of Mule Runtime: 3.8.7, 3.9.x, 4.1.x, and 4.2.x.
```

File: modules/ROOT/pages/anypoint-custom-metrics-connector.adoc (hunk 5 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/anypoint-custom-metrics-connector.adoc]:
```suggestion
* Mule Runtime 4.2.x, 4.1.x, 3.9.x, or 3.8.7
```

File: modules/ROOT/pages/configure-monitoring-cloudhub.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/configure-monitoring-cloudhub.adoc]: Which is a tag in the pom file.

**Pull request merged**